libs/cdk/utils/directives/focusable-list/focusable-list.directive.ts

```diff
@@ -40,6 +40,7 @@ import { ScrollPosition, scrollIntoView } from './scroll';
 
 export interface FocusableListItemFocusedEvent {
     index: number;
+    id: string | null;
```

Comment:
While I doubt this event is commonly dealt with by our users, this is a breaking change, which I think can be avoided by making the id the final property of this event and also optional (`?`)